packages/amplication-server/src/core/subscription/subscription.controller.ts

```diff
@@ -39,6 +45,15 @@ export class SubscriptionController {
           updateStatusDto.id,
           createSubscriptionInput
         );
+        if (createSubscriptionInput.plan === EnumSubscriptionPlan.Pro) {
+          await this.analyticsService.track({
+            userId: "unknown",
```

Comment:
can we pass the real user id?

Reply:
I would love to, but it doesn't arrive from webhook. I've contacted Stigg in order to find out if it's possible to pass user id.